tests/app/test_utilities.py

```diff
@@ -348,3 +349,9 @@ def test_add_to_details_with_internal_message_first(self):
 
             self.assertEqual(result[0], populated_message[0])
             self.assertEqual(result[1], populated_message[1])
+
+    def test_add_users_and_business_details_empty_messages_raises_Value_Error(self):
+        messages = None
+
+        with self.app.app_context(), self.assertRaises(ValueError):
```

Comment:
I don't think you need an app_context to test this?

Reply:
Will remove it 👍